sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Triggers/ServiceBusTriggerBindingStrategy.cs

```diff
@@ -53,8 +52,12 @@ public ServiceBusReceivedMessage[] BindMultiple(ServiceBusTriggerInput value, Va
             }
 
             var bindingData = new Dictionary<string, object>(StringComparer.OrdinalIgnoreCase);
-            SafeAddValue(() => bindingData.Add("MessageReceiver", value.Receiver));
-            SafeAddValue(() => bindingData.Add("MessageSession", value.SessionReceiver));
+            // TODO - investigate why the parameter names need to be hard-coded here but they are not
+            // for binding to sender
+            SafeAddValue(() => bindingData.Add("MessageReceiver", value.MessageActions));
```

Comment:
Is it worth supporting the old names if we change the type?

Reply:
I thought we wanted to do this.. but I don't think so.

Reply:
I think it makes things more confusing and doesn't allow us to add the MessageReceiver type back if required.

Reply:
I'm not sure I agree. I can see people going through working functions and just swapping out the types. I think we will have to decide on the Receiver before GA - I don't think it is something we would add post-GA. One other thing I need to investigate is why the binding requires a specific parameter name to begin with. For the sender, you can use any parameter name.

Reply:
Is it only the type? Do all method names and parameters match?

Reply:
I mean with track 1. If customers would need to change the parameter name and usages changing the name is not a problem.

Reply:
That is what I am proposing - that they wouldn't need to change the parameter name. Which name are you referring to? The Type name?